.circleci/config.yml

```
@@ -0,0 +1,20 @@
+version: 2.1
+
+jobs:
+  build-and-test:
+    docker:
+      - image: circleci/python:2.7
+    steps:
+      - checkout
+      - run: pip install .
```

Recently found out you can install the test dependencies based on `setup.py`'s extras.
```suggestion
      - run: pip install '.[test]'
```

---

Thanks, it worked and cleans up the config.